core/src/main/java/org/infinispan/xsite/ClusteredCacheBackupReceiver.java

```diff
@@ -126,9 +130,8 @@ public ClusteredCacheBackupReceiver(String cacheName) {
    public void start() {
       //it would be nice if we could inject bootstrap component
       //this feels kind hacky but saves 3 fields in this class
-      ComponentRegistry cr = cache.getAdvancedCache().getComponentRegistry();
-      TransactionHandler txHandler = new TransactionHandler(cache, cr.getTransactionTable());
-      defaultHandler = new DefaultHandler(txHandler, cr.getComponent(BlockingManager.class));
+      TransactionHandler txHandler = new TransactionHandler(cache, componentRegistry.getTransactionTable());
```

Comment:
can you revert this and remove the `componentRegistry' field? I know I'm complaining just to save 8 bytes eh eh

Reply:
Okay.